Feature/doc restructure micro ros agent (#16)
microROS/micro-ROS-doc: README.md

[![GitHub license](https://img.shields.io/github/license/microROS/micro-ROS-doc.svg)](https://github.com/microROS/micro-ROS-doc)
![GitHub release](https://img.shields.io/github/release/microROS/rosidl_typesupport_microxrcedds.svg?label=last_release)

> ![GitHub (pre-)release](https://img.shields.io/github/release-pre/microROS/rosidl_typesupport_microxrcedds.svg?label=pre_release)

Reviewer: Why are these shields pointing to typesupport repository?

Author: Type mistake.

Should be added release shields for each package?

# Overview 

> [MicroROS](https://microros.github.io/micro-ROS/) is a project that aims to puts ROS2 onto microcontrollers, making them first-class participants of the ROS 2 environment. 

Reviewer: ```suggestion
[micro-ROS](https://microros.github.io/micro-ROS/) is a project that aims to put ROS2 into microcontrollers, making them first-class participants of the ROS 2 environment. 
```

> This documentation is meant to help a developer to use MicroROS. 

Reviewer: ```suggestion
This documentation is a guide for the developer on the use micro-ROS. 
```

> It will describe all the functionalities of the different packages that make it up.

Reviewer: ```suggestion
It will describe all the functionalities of the different packages that micro-ROS is build on.
```

1. [Installation](Installation/README.md)
1. [Quick start](Quick_start/README.md)
1. [rmw_microxrcedds](rmw_microxrcedds/README.md)
1. [rosidl_typesupport_microxrcedds](rosidl_typesupport_microxrcedds/README.md)

> 1. [MicroROS Agent](MicroROS_Agent/README.md)

Reviewer: ```suggestion
1. [micro-ROS Agent](MicroROS_Agent/README.md)
```
